README.md

```diff
@@ -12,21 +14,23 @@ Free your self from distractions with our minimalists thought journal and note-t
 ./run.sh
 ```
 
-### Venv
+### Virtualenv
 ```bash
 python3 -m venv .venv
 pip install -r requirements.txt
 python3 src/application_entry.py
 ```
 ## How it works
 
-Running this program <how do they run the program $python src/application_entry.py?> Will turn your terminal
-into your very own ThoughtBox where you are free to take your notes however you like as long as its ascii approved.
-Built with python prompt toolkit you can navigate the entire program without a mouse.
-You can open the menu with ctr-H containing only the bare minimalist functionality.
-No more having to worry about centering or choosing the 'right' font anymore.
+Running this program will turn your terminal into your very own **Thought Box** where you are free to take your notes however you like as long as its ascii approved.
```

Comment:
Running this program will turn your terminal into your very own **Thought Box**, where the only limits on the notes you can take are the ASCII character set and your imagination.